health/health.d/swap.conf

```diff
@@ -25,11 +26,15 @@ component: Memory
 component: Memory
        os: linux freebsd
     hosts: *
-     calc: $used * 100 / ( $used + $free )
+     calc: ($used + $free) > 0 ? ($used * 100 / ($used + $free)) : 0
     units: %
     every: 10s
      warn: $this > (($status >= $WARNING)  ? (80) : (90))
      crit: $this > (($status == $CRITICAL) ? (90) : (98))
     delay: up 30s down 15m multiplier 1.5 max 1h
-     info: swap memory utilization
+     info: Percentage of used swap. \
+           High swap memory utilization. \
+           It affects the performance of your system, which can even become unresponsive. \
```

Comment:
No, it does not.

Actual swap IO affects system performance. having data in swap space has zero impact on performance. This distinction is _very_ important because it’s one that most users have trouble understanding, and because Linux does not preemptively remove pages from swap (IOW, until all refernces to a page are gone or Linux needs the space for another page, any given page will _not_ be removed from swap space).